src/lib/pgp-key.cpp

```diff
@@ -1472,8 +1472,8 @@ pgp_key_write_autocrypt(pgp_dest_t &dst, pgp_key_t &key, pgp_key_t &sub, size_t
             res = stream_write_key(&key.pkt, &memdst);
         }
 
-        res = res && stream_write_userid(&key.get_uid(uid).pkt, &memdst) &&
-              stream_write_signature(&cert->sig, &memdst);
+        key.get_uid(uid).pkt.write(memdst);
```

Comment:
Should it be `if (!res) { key.get_uid(uid).pkt.write(memdst); }`
to preserve original behaviour?

Reply:
Thanks, will update the PR. Actually it will be changed again in the next PRs, but let's keep things consistent.